4.0.0/migration.md

```diff
@@ -27,135 +146,148 @@ The migration process is initiated by the Installer if the following requirement
+        deploy.go:1685] waiting for mysql to postgres migration to finish... (attempt 2/3600)
+        deploy.go:1636] mysql to postgres migration finished correctly
+
+
+7. Ensure that collector (running at this point) and the API server (not running at this point, as it was scaled it to 0 replicas earlier) are deployed using the new manifests pointing to PostgreSQL:
+
+        deploy.go:1640] applying sysdigcloud-collector
+        deploy.go:1645] applying sysdigcloud-api
+        waitforpods.go:311] Waiting for deployment "sysdigcloud-collector" rollout to finish: 1 old replicas are pending termination...
+        waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
+        waitforpods.go:311] Waiting for deployment "sysdigcloud-collector" rollout to finish: 1 old replicas are pending termination...
+        waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
+        waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
+        waitforpods.go:311] Waiting for deployment "sysdigcloud-collector" rollout to finish: 1 old replicas are pending termination...
+        waitforpods.go:311] Waiting for deployment "sysdigcloud-collector" rollout to finish: 1 old replicas are pending termination...
+        waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
+        waitforpods.go:311] Waiting for deployment "sysdigcloud-collector" rollout to finish: 1 old replicas are pending termination...
+        waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
+        waitforpods.go:311] Waiting for deployment "sysdigcloud-collector" rollout to finish: 1 old replicas are pending termination...
+        waitforpods.go:311] Waiting for deployment "sysdigcloud-collector" rollout to finish: 1 old replicas are pending termination...
+        waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
+        waitforpods.go:318] deployment "sysdigcloud-collector" successfully rolled out
+        waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
+        waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
+        waitforpods.go:318] deployment "sysdigcloud-api" successfully rolled out
+        waitforpods.go:74] waitforpods(sysdigcloud-collector,sysdigcloud-api) completed in 1m32s
+
+
+8.  Ensure that collector block for the new agent connections is disabled (`GET /api/admin/agents/new/connections?block=false` HTTP request to API will be performed under the hood):
```

Comment:
Ensure that the collector block for the new agent connections is disabled. The `GET /api/admin/agents/new/connections?block=false` HTTP request to the API will be performed under the hood: